docs/root/source/timestamps.md

```diff
@@ -1,95 +0,0 @@
-# Timestamps in BigchainDB
```

Comment:
I think we still need these since we still add timestamps for blocks and voters (as well as when we validate a fulfillment, although I'm not sure if that's actually shown)?